src/index.ts

```diff
@@ -45,6 +45,7 @@ interface IConfig extends IStaticContext {
 interface IVariant {
     name: string;
     enabled: boolean;
+    feature_enabled?: boolean;
```

Comment:
Added this as optional so we don't break existing bootstraps. Can also make it required and release a new major. WDYT?